schema/validator.go

```diff
@@ -38,7 +41,7 @@ func (e ValidationError) Error() string {
 }
 
 // Validate validates the given reader against the schema of the wrapped media type.
-func (v Validator) Validate(src io.Reader) error {
+func (v Validator) Validate(src io.Reader, verbose bool, out *log.Logger) error {
```

Comment:
I assume that you mean 
```go
func output(verbose bool, out *logger, validate functype){
// todo
}
```
I mean a returned digests is needed from `validate functype`.